src/backend/distributed/utils/resource_lock.c

```diff
@@ -70,7 +99,7 @@ lock_shard_metadata(PG_FUNCTION_ARGS)
 		ereport(ERROR, (errmsg("no locks specified")));
 	}
 
-	/* we don't want random users to block writes */
+    /* we don't want random users to block writes */
```

Comment:
this is where I was puzzled.
my local citus_indent introduced those. I reverted them before submitting because they were irrelevant to what I was doing.

This time travis rejected it.  I fallen back to what citus_indent did.